File: tuan 2/ex5-1.js
let now = nowDate();

let hours = now.getHours();
let minutes = now.getMinutes();
let seconds = now.getSeconds();
console.log(`${hours}:${minutes}:${seconds}`);

let ms = now.getTime();
let days = Math.floor(ms / (1000 * 60 * 60 * 24));
console.log(days);

File: tuan 2/ex5-1.py
from datetime import datetime

now = datetime.now()

hours = now.hour
minutes = now.minute
seconds = now.second

print(f"{hours}:{minutes}:{seconds}")

ms = now.timestamp() * 1000

days = int(ms // (1000 * 60 * 60 * 24))
print(days)